### PR TITLE
Added parser for iosxe/show_wlan_summary

### LINKED ISSUE #51: 26. show wlan summary
```

ewlc1# show wlan summary

Number of WLANs: 4

ID   Profile Name                     SSID                             Status Security                                                                                             
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
17   lizzard_Global                  lizzard                         UP     [WPA2][802.1x][FT + 802.1x][AES],[FT Enabled]                                                        
18   wip_Global                      wip                             UP     [WPA2][802.1x + CCKM][AES]                                                                           
19   internet_Global                  internet                         UP     [open],MAC Filtering                                                                                 
20   lizzard-l_Global                lizzard-legacy                  UP     [WPA2][802.1x][AES]                                                                                  

```
```
ewlc2# show wlan summary
Number of WLANs: 3

ID   Profile Name                     SSID                             Status Security                                                                                             
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
17   lizzard_Fabric_F_cf6efda4       lizzard                         UP     [WPA2][FT + 802.1x][AES],[FT Enabled]                                                                
18   lizzard_l_Fabric_F_90c6fda4     lizzard_legacy                  UP     [WPA2][802.1x][AES]                                                                                  
19   internet_Fabric_F_ed7ae33f       internet                         UP     [open],MAC Filtering                                                                                 

```
```
ewlc3# show wlan summary 

Number of WLANs: 1

ID   Profile Name                     SSID                             Status Security                                                                                             
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
1    north                            north                            UP     [WPA2][802.1x][AES]                                                                                  

```
```
ewlc4# show wlan summary 
Number of WLANs: 3

ID   Profile Name                     SSID                             Status Security                                                                                             
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
17   lizzard_Fabric_F_cf6efda4       lizzard                         UP     [WPA2][FT + 802.1x][AES],[FT Enabled]                                                                
18   lizzard_l_Fabric_F_90c6fda4     lizzard_legacy                  UP     [WPA2][802.1x][AES]                                                                                  
19   internet_Fabric_F_ed7ae33f       internet                         UP     [open],MAC Filtering                                                                                 

```
```
ewlc5# show wlan summary 
Number of WLANs: 3

ID   Profile Name                     SSID                             Status Security                                                                                             
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
17   lizzard_Fabric_F_dee07354       lizzard                         UP     [WPA2][FT + 802.1x][AES],[FT Enabled]                                                                
18   lizzard_l_Fabric_F_90c6c2cd     lizzard_legacy                  UP     [WPA2][802.1x][AES]                                                                                  
19   internet_Fabric_F_ed7a62da       internet                         UP     [open],MAC Filtering                                                                                 

```

### PR DESCRIPTION
**Pull request by reustaquio22**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/tests/ShowWlanSummary/cli/equal/golden_output_output.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+Number of WLANs: 4
+
+ID   Profile Name                     SSID                             Status Security
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+17   lizzard_Global                  lizzard                         UP     [WPA2][802.1x][FT + 802.1x][AES],[FT Enabled]
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/tests/ShowWlanSummary/cli/equal/golden_output_output.txt:6]: This spacing seems off, can you confirm it is correct?

**Comment by itdependsnetworks** [comment]
After spacing check/update, open PR

File: src/genie/libs/parser/iosxe/tests/ShowWlanSummary/cli/equal/golden_output_output.txt (hunk 1 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/tests/ShowWlanSummary/cli/equal/golden_output_output.txt:6]: @itdependsnetworks I confirm, all examples in this issue are identical. 
[networktocode issue# 26](https://github.com/networktocode/genieparser/issues/51)

**Comment by itdependsnetworks** [comment]
submit pr

**Comment by itdependsnetworks** [comment]
Cisco PR https://github.com/CiscoTestAutomation/genieparser/pull/256 created

**Pull request merged**